staging/src/k8s.io/client-go/features/features.go

```diff
@@ -99,8 +101,23 @@ func AddFeaturesToExistingFeatureGates(registry Registry) error {
 //	// then replace client-go's feature gates implementation with your implementation
 //	clientgofeaturegate.SetFeatureGates(utilfeature.DefaultMutableFeatureGate)
 func SetFeatureGates(newFeatureGates Gates) {
```

Comment:
Stefan requested a rename to `ReplaceFeatureGates` I think.

Reply:
thanks, I missed it.